src/FSLibrary/MissionSimulatePubnet.fs

```diff
@@ -14,12 +14,17 @@ open StellarStatefulSets
 open StellarNetworkData
 open StellarSupercluster
 open StellarCoreHTTP
+open StellarKubeSpecs
 
 
 let simulatePubnet (context: MissionContext) =
     let context =
         { context with
-              coreResources = SimulatePubnetResources
+              coreResources =
+                  if context.networkSizeLimit <= SimulatePubnetNetworkSizeThreshold then
```

Comment:
should we just let SimulatePubnetResources figure out the right resources based on the number of nodes?

Reply:
this will simplify the code. and also you can just scale the regular resources by some factor instead of hardcoding it

Reply:
Makes sense! Updated the PR :)